### ISSUE TITLE
Hang on launch

### ISSUE THREAD

**Issue by zmingee** [martanne/dvtm#83]
When I launch `dvtm`, it just seems to hang forever and causes 100% CPU load. The screen goes blank with a cursor in the corner like normal, but then just stays like that.

```
[xerxes:dvtm]$ ./dvtm -v
dvtm-v0.15-37-gb45828d-dirty © 2007-2016 Marc André Tanner
```

The following are the only changes I've made:
```
diff --cc config.def.h
index 513c734,9e47e2e..0000000
--- a/config.def.h
+++ b/config.def.h
@@@ -69,7 -74,10 +77,7 @@@ static Layout layouts[] =
        { "[ ]", fullscreen },
  };

 -<<<<<<< Updated upstream
--#define MOD  CTRL('g')
 -=======
+ #define MOD  CTRL('a')
  #define TAGKEYS(KEY,TAG) \
        { { MOD, 'v', KEY,     }, { view,           { tags[TAG] }               } }, \
        { { MOD, 't', KEY,     }, { tag,            { tags[TAG] }               } }, \
@@@ -140,6 -168,8 +148,7 @@@ static KeyBinding bindings[] =
        TAGKEYS( '3',                              2)
        TAGKEYS( '4',                              3)
        TAGKEYS( '5',                              4)
+       TAGKEYS( '6',                              5)
 ->>>>>>> Stashed changes
  };

  static const ColorRule colorrules[] = {
```

If I use the packaged version of `dvtm` available on the Arch Linux official repositories it works fine. However, if I attempt to compile it from source (or even use the `dvtm-git` pacakage on the AUR) it just crashes and hangs.

```
[xerxes:dvtm]$ uname-a
Linux xerxes 4.15.7-1-ARCH #1 SMP PREEMPT Wed Feb 28 19:01:57 UTC 2018 x86_64 GNU/Linux

[xerxes:dvtm]$ gcc -v
Using built-in specs.
COLLECT_GCC=/usr/bin/gcc
COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-pc-linux-gnu/7.3.0/lto-wrapper
Target: x86_64-pc-linux-gnu
Configured with: /build/gcc/src/gcc/configure --prefix=/usr --libdir=/usr/lib --libexecdir=/usr/lib --mandir=/usr/share/man --infodir=/usr/share/info --with-bugurl=https://bugs.archlinux.org/ --enable-languages=c,c++,ada,fortran,go,lto,objc,obj-c++ --enable-shared --enable-threads=posix --enable-libmpx --with-system-zlib --with-isl --enable-__cxa_atexit --disable-libunwind-exceptions --enable-clocale=gnu --disable-libstdcxx-pch --disable-libssp --enable-gnu-unique-object --enable-linker-build-id --enable-lto --enable-plugin --enable-install-libiberty --with-linker-hash-style=gnu --enable-gnu-indirect-function --enable-multilib --disable-werror --enable-checking=release --enable-default-pie --enable-default-ssp
Thread model: posix
gcc version 7.3.0 (GCC)

```

**Comment by zmingee** [comment]
Also, for the record, everything was working fine. Then I rebooted and the issues started.

**Comment by MarSoft** [comment]
Same issue here, also after reboot. I have decided to migrate back to tmux, because this issue in addition to #73 and #74 makes dvtm unusable.

**Comment by Larivact** [comment]
Might be related to #84, does setting `TERM=dvtm-256color` before starting dvtm solve the issue?

**Comment by zmingee** [comment]
I just tried on a fresh build with the TERM variable and the issue is still present.

For now I'm using `tmux`. I may take a look at the `dvtm` code when I get time and see if I can figure it out.

**Comment by shua** [comment]
PR'd a fix in #85. @zmingee Can you verify whether this fixes it for you?
